packages/block-editor/src/components/rich-text/native/test/index.native.js

```diff
@@ -83,11 +88,11 @@ describe( '<RichText/>', () => {
 		} );
 	} );
 
-	describe( 'when changes arrive from Aztec', () => {
```

Comment:
I renamed it mainly to make the section more open to other test cases like the one added in the PR. The formatting change was mainly driven by using the same we have in the other `describe` section:
https://github.com/WordPress/gutenberg/blob/307e15e28856d318084d469d29c851a57ffb30c8/packages/block-editor/src/components/rich-text/native/test/index.native.js#L148

In any case, I don't have a strong opinion on the formatting. We could still use `when...`, but if prefer this option, it would be great to make it consistent across the file.

Reply:
Thanks @dcalhoun for elaborating on your response 🙇 !

> I am all for consistency. That said, at this time, I would argue that we _do not have_ a consistent approach established. For this specific example, even `Font Size` and `Value changes` differ on casing. Admittedly, casing is not an important thing by any means, but I figure it is relevant nonetheless.

Ah, good point. I'll update the description to match the casing.

> My original goal with [my inquiry](https://github.com/WordPress/gutenberg/pull/57652#discussion_r1445323016) was to avoid a situation where we (contributors) swap code back and forth. Without automated enforcement via a linter, it is difficult to ensure consistency. There are [examples of the "when" formatting](https://github.com/search?q=repo%3AWordPress%2Fgutenberg%20describe(%20%27when&type=code) in the code base as well. That is not to say we must adhere to that, but, again, I inquired to understand the rationale for modifying this one location.
>
> For context, I was taught the "when" pattern as an approach to make test output read as a sentence. I.e., `[subject] when [context] should [expectation]`. Again, not advocating that we adhere to this format, merely sharing my rationale.

Yep, we don't have a clear guideline about the `describe` section. I checked the [Gutenberg documentation about testing](https://developer.wordpress.org/block-editor/contributors/code/testing-overview/#describing-tests) but there's no explicit reference to the `describe` statement, apart from the following:
> Use a describe block to group test cases.

Using `when`, although a bit more verbose, I agree it's a format more aligned with the description in test cases. I'll update it for this file. Thanks 🙇 !

Reply:
> Using `when`, although a bit more verbose, I agree it's a format more aligned with the description in test cases. I'll update it for this file. Thanks 🙇 !

I applied this suggestion in https://github.com/WordPress/gutenberg/pull/57652/commits/870a99af5e20ea6af222ac2a301499edbd7046c1.